src/components/binding/binding-loader.js

```diff
@@ -254,9 +275,9 @@ genotet.BindingLoader.prototype.loadBindingList = function() {
     type: 'list-binding'
   };
   this.get(genotet.data.serverURL, params, function(data) {
-    genotet.data.bindingGenes = [];
+    genotet.data.bindingGenes = {};
     data.forEach(function(dataInfo) {
-      genotet.data.bindingGenes.push(dataInfo.gene);
+      genotet.data.bindingGenes[dataInfo.gene] = dataInfo.fileName;
```

Comment:
We use fileName as binding data keys. genes are descriptive names which are not guaranteed to be primary keys.